### PR TITLE
[query] import_google_sheet

### PR DESCRIPTION
**Pull request by Aleisha02**

### CODE REVIEW

File: hail/python/hail/methods/impex.py
Comment (danking) [inline hail/python/hail/methods/impex.py:3013]: The spreadsheet id can only be a str, not a list of str, so let's change this to: 
```
spreadsheetID: :class:`str`
```

File: hail/python/hail/methods/impex.py
Comment (danking) [inline hail/python/hail/methods/impex.py:3014]: Let's describe this as the spread sheet id. Can you include an example above of what a Google Sheet URL and the corresponding ID looks like? I think most people will have trouble finding the ID on the first try.

File: hail/python/hail/methods/impex.py
Comment (danking) [inline hail/python/hail/methods/impex.py:3044]: Can you delete this empty line at the end of the file?

File: hail/src/main/scala/is/hail/expr/ir/StringTableReader.scala
Comment (danking) [inline hail/src/main/scala/is/hail/expr/ir/StringTableReader.scala:160]: Can you delete this empty line?

File: hail/src/main/scala/is/hail/expr/ir/TableIR.scala
Comment (danking) [inline hail/src/main/scala/is/hail/expr/ir/TableIR.scala:3003]: Can you move this back to around line 102?

**Review by danking — DISMISSED** [review]
This looks awesome. I just have a comment about the docs and a few nitpicks.

**Comment by Aleisha02** [comment]
Ready for another look

**Review by danking — APPROVED** [review]
LGTM!

**Comment by danking** [comment]
grr I think this betrothed.

**Comment by danking** [comment]
bitrotted*

**Comment by danking** [comment]
Waiting on google to explain why the latest version of the sheets library gives me 403s.

**Comment by danking** [comment]
No explanation from Google, but I figured out how to make it work. TLDR: don't use the thing that seems like the obviously correct choice, use some obscure other approach.

**Comment by danking** [comment]
@tpoterba I heavily modified this so best to get a second review.

**Review by danking — APPROVED** [review]

**Pull request closed without merging**
